clib/src/main/scala/scala/scalanative/libc/signal.scala

```diff
@@ -7,7 +7,9 @@ import scalanative.unsafe._
 object signal {
 
   // Signals
-
+  @deprecated(
+    "Not in C standard, use posix signal. To be removed in next version",
```

Comment:
re: "Not in C standard".  Which C standard C89, C11, ISO C?
There are any number of C standards in play.

I think you mean "IEEE Std 1003.1™-2017 (Revision of IEEE Std 1003.1-2008)". You/we probably only want to give the portion to
the left of the open parenthesis. 

Yes, picky, but I think lack of precision here is an indicator of 
`clib` not having a `00_READ_ME` which gives a clear statement
of which standard files in that project are to follow.   Not to try to 
solve the general `clib` problem here, but I think there is an 
opportunity to remove some technical debt from `clib`.

Reply:
The function it not in any C standard - I think the intent here is just let the deprecation point the user to the correct place, This is really only for a UNIX user who may have used this function from `clib`. I can change the deprecation warning and I probably should say what version it will be removed in or deprecated in.

Reply:
Ok, I updated those too, hopefully to your liking.